include/aikido/control/KinematicSimulationTrajectoryExecutor.hpp

```diff
@@ -58,6 +58,8 @@ class KinematicSimulationTrajectoryExecutor : public TrajectoryExecutor
   /// Trajectory's MetaSkeletonStateSpace
   statespace::dart::ConstMetaSkeletonStateSpacePtr mStateSpace;
 
+  ::dart::dynamics::MetaSkeletonPtr mMetaSkeleton;
```

Comment:
Nit: missing documentation. I believe that this is the controlled subset of `mSkeleton` for the currently executing trajectory.